backend/app/Http/Requests/CreateServiceCategoryRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateServiceCategoryRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'title' => 'required|string',
+            'order' => 'integer',
```

Comment:
Im not quite sure if  'order' attribute is necessary. Do u mind if I delete it?

Reply:
leave it, we will think about it a bit later